### ISSUE TITLE
crash on startup

### ISSUE THREAD

**Issue by cursinorock** [TheExceptionist/Coherent-Villages#17]
Mod ver: coherent_villages_beta_v1.1.9

Crash when starts forge loads.
[crash-2018-09-27_21.20.21-client.txt](https://github.com/TheExceptionist/Coherent-Villages/files/2426299/crash-2018-09-27_21.20.21-client.txt)

**Comment by SaucissePuree** [comment]
IDEM
Crash when starts forge loads.
[coherent_villages_beta_v1.1.9.txt](https://github.com/TheExceptionist/Coherent-Villages/files/2426890/coherent_villages_beta_v1.1.9.txt)

**Comment by Njet75** [comment]
Same for me 1.1.5 works

**Comment by xannor** [comment]
Just to add my exp, I got this error too, but deleting the config does not reproduce it so I believe it is due to configuration changes. since your config seems to be in flux, you might want to version it, storing the version first, so you can adapt and adjust when you have breaking changes to it.

I'll also note that I was going from the mislabeled 1.0 to the current 1.29, and had to confirm two missing entities, evil villager and creeper hunter, so I do not know if that change was the culprit.

**Comment by TheExceptionist** [comment]
No, were you running on a world that was using 1.0? Because I deleted those entities

**Comment by TheExceptionist** [comment]
Hopefully fixed
Tell me if you guys still have any problems

**Comment by cursinorock** [comment]
work here :3
